shared/lib/delegation/caveatBuilder/specificActionERC20TransferBatchBuilder.ts

```diff
@@ -0,0 +1,67 @@
+import { encodePacked } from '@metamask/abi-utils';
+import { Hex, bytesToHex } from '@metamask/utils';
+import { Caveat } from '../caveat';
+import { DeleGatorEnvironment } from '../environment';
+import { isAddress } from '../utils';
+
+export const specificActionERC20TransferBatch =
+  'specificActionERC20TransferBatch';
+
+export function specificActionERC20TransferBatchBuilder(
+  environment: DeleGatorEnvironment,
+  erc20TokenAddress: Hex,
```

Comment:
For maintainability, can we match the exact terms name from the contract?

https://github.com/MetaMask/delegation-framework/blob/main/src/enforcers/SpecificActionERC20TransferBatchEnforcer.sol

So `tokenAddress`, `recipient`, `amount`, `firstTarget`, and `firstCallData`?